### DISCUSSION TITLE
Is the app abandoned?

### DISCUSSION THREAD

**Discussion by valsaven** [MobiusSync/MobiusSync#96] [Q&A] [answered]
Hi!

Thank you for the handy application, but lately it keeps getting the error "Insufficient space on disk for database" which makes it impossible to use.

Also, the latest version of Syncthing is v1.23.0 and on Twitter, the latest news for January 2022.

Do you have plans to develop the app further or has it been abandoned?

**Comment by kuochang** [comment]
I got a similar error and this struggled me for a while. However, today I released around 10GB iPhone space(by deleting photos and videos) and the error is gone. It is weird at the first place since my disk space should be way enough before I deleted the photos/videos… Anyway, it works fine now. 

BTW, my iOS and iPad OS are v16.2.

**Comment by tohvanah** [comment]
There's a global and folder specific setting for minimum disk space. Check both against your actual free space on device. 

In any case, I hope this app is not abandoned! There was an App Store update 3 months ago. I recently purchased it.

**Comment by kuochang** [comment]
Thanks, tohbanah! I found the minimum disk space setting in global and folder. It’s helpful.:)

BTW, I am the paid user too and I do hope the app is maintained.:)

**Comment by SimonPickup** [comment, marked as answer]
The app is not abandoned - but it is a side project without significant development resources.

As for the "insufficient space" issue, this sounds like #17.

**Reply by danielraffel** [reply]
Would you please consider making this project open source? The app hasn't received updates for more than a year and it probably hasn't been a significant source of revenue either, opening it up could allow a wider community to contribute to its maintenance. And, it would be a nice gift to the syncthing community. I know you've considered it in the past, thanks for reconsidering.

**Reply by YouRexcellent** [reply]
Any plans on releasing a version that gets past [this CVE](https://www.cvedetails.com/cve/CVE-2022-46165/)?

Many thanks for this great tool.  Hope it gets the attention it deserves.

**Reply by YouRexcellent** [reply]
Thank you for the updates to the app. Pro version purchased.
